scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -304,6 +305,33 @@ def get_project(self):
             sys.exit(1)
         print('Project id: %s' % self.project_id)
 
+    def check_billing_enabled(self):
+        """Check if billing is enabled."""
+        return_code, out, err = self._run_command(
+            ['gcloud', 'alpha', 'billing', 'projects', 'describe',
+             self.project_id, '--format=json'])
+        if return_code:
+            print(err)
+            self._billing_not_enabled()
+        try:
+            billing_info = json.loads(out)
+            if billing_info.get('billingEnabled'):
+                print('Billing IS enabled.')
+            else:
+                self._billing_not_enabled()
+        except ValueError:
+            self._billing_not_enabled()
+
+    def _billing_not_enabled(self):
+        """Print message and exit."""
+        print('\nIt seems that billing is not enabled for your project. '
+              'You can check whether billing has been enabled in the '
```

Comment:
nit: A message about re-running once billing is enabled would be nice.

Reply:
Done